### PR TITLE
Transfer multiple maps support

### PR DESCRIPTION
**Pull request by xluox**
I have made a new branch and transferred some changes from the old PO multiple map branch into the new branch.

### CODE REVIEW

**Comment by xluox** [comment]
@vwxyzjn How to change the pointer of microrts?

**Comment by xluox** [comment]
@vwxyzjn I want to focus on supporting multiple maps for evaluation first.

File: experiments/league.py [hunk 1]
```diff
@@ -375,40 +377,41 @@ def get_leaderboard_existing_ais(existing_ai_names):
...
-                        loss=int(item == -1),
-                    ).save()
+                for match in range(args.num_matches // 2):
+                    m = Match(args.partial_obs, match_up, args.maps[match])
+                    challenger = AI.get_or_none(name=m.p0)
+                    defender = AI.get_or_none(name=m.p1)
+
+                    r = m.run(1)
+                    for item in r:
+                        drawn = False
```
[the remark this answers is not in the document]
Comment (xluox) [reply experiments/league.py]: I was looping in num_matches, so if num_matches is 10, then run each map once. But we can definitely do your suggested way, which will do 10 matches for each map.

**Comment by vwxyzjn** [comment]
> @vwxyzjn How to change the pointer of microrts? 

```
cd gym_microrts/microrts
git checkout bf077a203e10329e0b23f01665d02eb5cfdc77fe
cd ..
cd ..
bash build.sh
# add the repository change to git.
```

> @vwxyzjn I want to focus on supporting multiple maps for evaluation first.

This sounds good.

**Review by vwxyzjn — APPROVED** [review]
LGTM

**Pull request merged**